<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To Redshift</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="arnoldToLambertButton">
      <property name="text">
       <string>Arnold to Lambert</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="mayaToRedshiftButton">
      <property name="text">
       <string>Maya to Redshift</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="arnoldToRedshiftButton">
      <property name="text">
       <string>Arnold to Redshift</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="redshiftToLambertButton">
      <property name="text">
       <string>Redshift to Lambert</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="selectButton">
        <property name="toolTip">
         <string>Select all arnolds</string>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="convertButton">
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
